build.gradle.kts

```diff
@@ -1,8 +1,23 @@
 plugins {
     java
+    application
     kotlin("jvm") version "1.9.23"
 }
 
+application {
+    // NOTE: In case you choose a different main class, make sure to update this path
+    mainClass = "KotlinApplicationKt"
+    // mainClass = "JavaApplication"
```

Comment:
cc @Ch4s3r Added the second option.

Reply:
I would add a note to the README about this configuration as well.

Reply:
Added